go-controller/pkg/config/const.go

```diff
@@ -0,0 +1,29 @@
+package config
+
+const (
+	// config.OVNClusterRouter is the name of the distributed router
+	OVNClusterRouter = "ovn_cluster_router"
+
+	JoinSwitchPrefix             = "join_"
+	ExternalSwitchPrefix         = "ext_"
+	GWRouterPrefix               = "GR_"
+	RouterToSwitchPrefix         = "rtos-"
+	SwitchToRouterPrefix         = "stor-"
+	JoinSwitchToGwRouterPrefix   = "jtor-"
+	GWRouterToJoinSwitchPrefix   = "rtoj-"
+	DistRouterToJoinSwitchPrefix = "dtoj-"
+	JoinSwitchToDistRouterPrefix = "jtod-"
+	EXTSwitchToGwRouterPrefix    = "etor-"
+	GWRouterToExtSwitchPrefix    = "rtoe-"
+
+	NodeLocalSwitch = "node_local_switch"
+
+	// priority of logical router policies on the OVNClusterRouter
+	EgressFirewallStartPriority           = "10000"
```

Comment:
Silly question: why are these strings? Just to make interpolation easier?

Reply:
My understanding is that there are strings because for the most part they are only used to be put in match strings for ovn commands, it is only egressFirewall that calculates anything from these values